backend/app/services/auth.py

```
@@ -1,145 +1,43 @@
+    """
+    user_service = UserService(db)
+    user_data = {
+        "email": "base@example.com",
+        "first_name": "Base",
+        "last_name": "User"
+    }
+    user = user_service.get_or_create_user(user_data)
+    return user
 
 def get_optional_current_user(
     credentials: Optional[HTTPAuthorizationCredentials] = Depends(security),
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)